convert_fendl.py

```diff
@@ -41,90 +42,308 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                    'N015mc.zip',  'O016mc.zip',  'F019mc.zip',  'Na023mc.zip',
+                    'Mg000mc.zip', 'Al027mc.zip', 'Si028mc.zip', 'Si029mc.zip',
+                    'Si030mc.zip', 'P031mc.zip',  'S000mc.zip',  'Cl035mc.zip', 
+                    'Cl037mc.zip', 'K000mc.zip',  'Ca000mc.zip', 'Ti046mc.zip', 
+                    'Ti047mc.zip', 'Ti048mc.zip', 'Ti049mc.zip', 'Ti050mc.zip', 
+                    'V000mc.zip',  'Cr050mc.zip', 'Cr052mc.zip', 'Cr053mc.zip', 
+                    'Cr054mc.zip', 'Mn055mc.zip', 'Fe054mc.zip', 'Fe056mc.zip', 
+                    'Fe057mc.zip', 'Fe058mc.zip', 'Co059mc.zip', 'Ni058mc.zip', 
+                    'Ni060mc.zip', 'Ni061mc.zip', 'Ni062mc.zip', 'Ni064mc.zip', 
+                    'Cu063mc.zip', 'Cu065mc.zip', 'Ga000mc.zip', 'Zr000mc.zip', 
+                    'Nb093mc.zip', 'Mo092mc.zip', 'Mo094mc.zip', 'Mo095mc.zip', 
+                    'Mo096mc.zip', 'Mo097mc.zip', 'Mo098mc.zip', 'Mo100mc.zip', 
+                    'Sn000mc.zip', 'Ta181mc.zip', 'W182mc.zip',  'W183mc.zip',  
+                    'W184mc.zip',  'W186mc.zip',  'Au197mc.zip', 'Pb206mc.zip', 
+                    'Pb207mc.zip', 'Pb208mc.zip', 'Bi209mc.zip'],
+            'file_type': 'ace',
+            'ace_files': ace_files_dir.glob('*.ace'),
+            'compressed_file_size': 100,
+            'uncompressed_file_size': 600
+        },
+        'photon':{
+            'base_url': 'https://www-nds.iaea.org/fendl21/fendl21e/',
+            'files': ['FENDLEP.zip'],
+            'file_type': 'endf',
+            'photo_files': endf_files_dir.glob('*.endf'),
+            'compressed_file_size': 2,
+            'uncompressed_file_size': 5
+        }
     }
 }
 
+compressed_file_size, uncompressed_file_size = 0, 0
+for p in ('neutron', 'photon'):
+    if p in args.particles:
+        compressed_file_size += release_details[args.release][p]['compressed_file_size']
+        uncompressed_file_size += release_details[args.release][p]['uncompressed_file_size']
+
 download_warning = """
-WARNING: This script will download {} of data.
-Extracting and processing the data requires {} of additional free disk space.
-""".format(release_details[args.release]['compressed_file_size'],
-           release_details[args.release]['uncompressed_file_size'])
+WARNING: This script will download {} MB of data.
+Extracting and processing the data requires {} MB of additional free disk space.
+""".format(compressed_file_size, uncompressed_file_size)
 
 # ==============================================================================
 # DOWNLOAD FILES FROM IAEA SITE
 
 if args.download:
     print(download_warning)
-    for f in release_details[args.release]['files']:
-        download(urljoin(release_details[args.release]['base_url'], f),
-                 as_browser=True, context=ssl._create_unverified_context())
+
+    for particle in args.particles:
+        if args.release == '2.1':
```

Comment:
nice idea to have a download folder, I think we can roll this out to other scripts in the future. The number of compressed files we end up with in the data folder gets a bit excessive when downloading lots of libraries. Can put the .zip files into such a folder for all releases in the fendl script?

Reply:
this will help remove lots of the if args.release=2.1 cases

Reply:
Ok, currently the download path is named based on the release. At what level should the download folder be general? Would you like the download folder to be something like fendl-download instead which would imply that only FENDL data would go in there, or just call it something like downloads, so that other scripts could then use the same folder.

Reply:
In either case, it may still be worth having an extra folder for the 2.1 release as the ace files don't have any indication in the name that they are from FENDL. I think they clutter the directory and it is possible that they could clash with other libraries due to the generic naming.

Reply:
> Ok, currently the download path is named based on the release. At what level should the download folder be general? Would you like the download folder to be something like fendl-download instead which would imply that only FENDL data would go in there, or just call it something like downloads, so that other scripts could then use the same folder?

Based on release like you have is perfect for me